custom.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/grupozap/aegir/internal/pkg/utils"
+	livr "github.com/k33nice/go-livr"
+)
+
+func neq(args ...interface{}) livr.Validation {
+	not_allowed := utils.FirstArg(args...)
+
+	return func(value interface{}, builders ...interface{}) (interface{}, interface{}) {
+		if value == nil || value == "" {
+			return nil, nil
+		}
+
+		switch value.(type) {
+		case float64, string, bool:
+		default:
+			return nil, errors.New("FORMAT_ERROR")
+		}
+		if fmt.Sprint(value) == fmt.Sprint(not_allowed) {
+			return nil, errors.New("NOT_ALLOWED_VALUE")
+		}
+
+		return value, nil
+	}
+}
+
+func not_like(args ...interface{}) livr.Validation {
+	var re *regexp.Regexp
+	var flags string
+	if len(args) > 0 {
+		if len(args) > 1 {
+			if v, ok := args[1].(string); ok {
+				if v == "i" {
+					flags = "(?i)"
+				}
+			}
+		}
+
+		if v, ok := args[0].(string); ok {
+			reg, err := regexp.Compile(flags + v)
+			if err != nil {
+				re = regexp.MustCompile(".*")
```

Comment:
In this case, I think MustCompile isn't the best approach, this function throws panic. The best way is to put it in a package variable or a global variable: https://golang.org/pkg/regexp/#MustCompile